modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -117,92 +117,35 @@ public static class Sql {
         /** Session not found error. */
         public static final int SESSION_NOT_FOUND_ERR = SQL_ERR_GROUP.registerErrorCode(2);
 
-        /** Invalid query error. */
-        public static final int QUERY_INVALID_ERR = SQL_ERR_GROUP.registerErrorCode(3);
-
         /** Query without a result set error. */
         public static final int QUERY_NO_RESULT_SET_ERR = SQL_ERR_GROUP.registerErrorCode(4);
 
-        /** Missing primary key error. */
-        public static final int PRIMARY_KEY_MISSING_ERR = SQL_ERR_GROUP.registerErrorCode(5);
-
-        /** Multiple primary keys error. */
-        public static final int PRIMARY_KEYS_MULTIPLE_ERR = SQL_ERR_GROUP.registerErrorCode(6);
-
         /** Schema not found. */
-        public static final int SCHEMA_NOT_FOUND_ERR = SQL_ERR_GROUP.registerErrorCode(7);
-
-        /** Storage engine not valid. */
-        public static final int STORAGE_ENGINE_NOT_VALID_ERR = SQL_ERR_GROUP.registerErrorCode(8);
+        public static final int SCHEMA_NOT_FOUND_ERR = SQL_ERR_GROUP.registerErrorCode(5);
 
         /** Cursor is already closed error. */
-        public static final int CURSOR_CLOSED_ERR = SQL_ERR_GROUP.registerErrorCode(9);
-
-        /** Constraint violation: some keys can't be inserted because they violate unique constraint PK. */
-        public static final int DUPLICATE_KEYS_ERR = SQL_ERR_GROUP.registerErrorCode(10);
-
-        /** Constraint violation: deleting a column that belongs to the index. */
-        public static final int DROP_IDX_COLUMN_CONSTRAINT_ERR = SQL_ERR_GROUP.registerErrorCode(11);
-
-        /**  Too many grouping expressions. */
-        public static final int TOO_MANY_GROUPING_EXPRESSIONS_ERR = SQL_ERR_GROUP.registerErrorCode(12);
-
-        /** Unsupported sql operation. */
-        public static final int UNSUPPORTED_SQL_OPERATION_KIND_ERR = SQL_ERR_GROUP.registerErrorCode(13);
-
-        /** Unsupported DDL operation. */
-        public static final int UNSUPPORTED_DDL_OPERATION_ERR = SQL_ERR_GROUP.registerErrorCode(14);
-
-        /** Query validation error. */
-        public static final int QUERY_VALIDATION_ERR = SQL_ERR_GROUP.registerErrorCode(15);
+        public static final int CURSOR_CLOSED_ERR = SQL_ERR_GROUP.registerErrorCode(6);
 
-        /** Object is not found in schema. */
-        public static final int OBJECT_NOT_FOUND_ERR = SQL_ERR_GROUP.registerErrorCode(16);
+        /** Parsing error. */
+        public static final int PARSE_ERR = SQL_ERR_GROUP.registerErrorCode(7);
 
-        /** Object already exists in schema. */
-        public static final int OBJECT_ALREADY_EXISTS_ERR = SQL_ERR_GROUP.registerErrorCode(17);
+        /** Validation error. */
+        public static final int VALIDATION_ERR = SQL_ERR_GROUP.registerErrorCode(8);
```

Comment:
```suggestion
        public static final int QUERY_VALIDATION_ERR = SQL_ERR_GROUP.registerErrorCode(8);
```

Reply:
Do not have strong option on this but QUERY is unnecessary because SQL engine executes both queries and commands that do not produce results.

Reply:
It's not clear what validation do you mean. Query validation phase or session validation or user parameters validation or whatever?

Reply:
I see neither user parameter, nor session validation at the moment. I can can rename both `PARSE_ERR` and `VALIDATION_ERR` to `STMT_PARSE_ERR`, `STMT_VALIDATION_ERR`.

Reply:
Fixed.